cli/tests/pcluster/config/test_validators.py

```diff
@@ -937,3 +937,30 @@ def test_ebs_settings_validator(mocker, cluster_section_dict, ebs_section_dict,
 def test_shared_dir_validator(mocker, section_dict, expected_message):
     config_parser_dict = {"cluster default": section_dict}
     utils.assert_param_validator(mocker, config_parser_dict, expected_message)
+
+
+@pytest.mark.parametrize(
+    "cluster_section_dict, dcv_section_dict, expected_message",
+    [
+        (
+            {"base_os": "alinux", "dcv_settings": "dcv"},
+            {"enable": "master"},
+            "Please double check the 'base_os' configuration parameter",
+        ),
+        (
+            {"base_os": "centos6", "dcv_settings": "dcv"},
+            {"enable": "master"},
+            "Please double check the 'base_os' configuration parameter",
+        ),
+        (
+            {"base_os": "ubuntu1604", "dcv_settings": "dcv"},
+            {"enable": "master"},
+            "Please double check the 'base_os' configuration parameter",
+        ),
+        ({"base_os": "centos7", "dcv_settings": "dcv"}, {"enable": "master"}, None),
+        ({"base_os": "ubuntu1804", "dcv_settings": "dcv"}, {"enable": "master"}, None),
+    ],
+)
+def test_dcv_enabled_validator(mocker, cluster_section_dict, dcv_section_dict, expected_message):
```

Comment:
You can avoid to have `dcv_section_dict` and `cluster_section_dict` as a test parameters, since they are not changing.
The only real parameter for the test is the `base_os`.